WordPress/Classes/ViewRelated/Reader/Select Interests/Mock Data/InterestsDataSource.swift

```diff
@@ -0,0 +1,52 @@
+import Foundation
+
+class ReaderInterest {
```

Comment:
I agree, but since this code is just for testing purposes and is going to be removed once the network side of things are done later this week I'm going to leave it as it is.